### PR TITLE
NEW: 88190 add new task display fields and update task api

### PR DESCRIPTION
**Pull request by jessicamoorthy**
- Added security team email, time to complete, time to review fields
- Added logic to display if a task can create other tasks
- Updated task api

### CODE REVIEW

File: src/Model/Task.php [hunk 1]
```diff
@@ -1085,4 +1087,34 @@ public static function export_record($task)
...
+                    !empty($answerActionFields['TaskIDs'])) {
+                    return true;
+                }
+            }
+        }
+
+        // handle tasks that are visible,
+        // but content for a related task is generated after completion of the first task
+        // e.g. Security Risk Assessment
+        if ($this->TaskType == 'risk questionnaire' ||
```
Comment (roopamjain01) [inline src/Model/Task.php:1112]: why it is always true for rsk questionnaire and selection task?

Comment (roopamjain01) [reply src/Model/Task.php:1112]: Could you please rethink about the logic and check for all task type again, "getQuestionsData" funtion is written foe different purpose.

Comment (elliot-sawyer) [reply src/Model/Task.php:1112]: This all looks fine to me, I'm not sure how else to determine if a tasktype is a risk questionnaire. Lacking context I have nothing new to add, so I'll leave to @roopamjain01 to resolve

**Comment by elliot-sawyer** [comment]
I've resolved a merge conflict caused by the merger of another PR

**Pull request merged**